components/dashboard/src/settings/Account.tsx

```diff
@@ -7,23 +7,40 @@
 import { User } from "@gitpod/gitpod-protocol";
 import { useContext, useState } from "react";
 import Modal from "../components/Modal";
+import { getGitpodService } from "../service/service";
 import { UserContext } from "../user-context";
 import { SettingsPage } from "./SettingsPage";
 
 export default function Account() {
     const { user } = useContext(UserContext);
 
     const [modal, setModal] = useState(false);
+    const [typedEmail, setTypedEmail] = useState('');
+
+    const primaryEmail = User.getPrimaryEmail(user!);
+
+    const deleteAccount = async () => {
+        await getGitpodService().server.deleteAccount();
+        //TODO we are sad to see you go view
+        document.location.href = 'https://www.gitpod.io';
```

Comment:
independent of the TODO, I think we should use the website URL from branding.

Reply:
also this https://github.com/gitpod-io/gitpod/pull/3551#issuecomment-804763633

Reply:
We don't support "branding" anymore, just saas and self-hosted. In both cases users could land on our website.
But I changed this now to just go to logout.